### PR TITLE
pkgdiff: add more files for --build-system

### PR DESCRIPTION
**Pull request by thesamesam**

### CODE REVIEW

File: pkgdiff [hunk 1]
```diff
@@ -29,7 +29,7 @@ cat2=$(get_category "${2}")
 fn1=${1##*/}
 fn2=${2##*/}
 
-export PORTAGE_TMPDIR=/tmp/pkgdiff
+export PORTAGE_TMPDIR=$(mktemp --tmpdir -d pkgdiff.XXXXXX)
```
Comment (mgorny) [inline pkgdiff:32]: I don't like this. Right now we're reusing the same directory, so it's easy to inspect stuff manually and clean it up afterwards.

File: pkgdiff [hunk 2]
```diff
@@ -42,13 +42,25 @@ s2=$(sed -nr 's/^declare -x S="(.*)"/\1/p' "${PORTAGE_TMPDIR}"/portage/${cat2}/$
...
+						-name 'Gemfile' -o \
+						-name 'Gemfile.lock' -o \
+						-name '*.opam' -o \
+						-name 'dune' -o \
+						-name 'dune-project' -o \
+						-name 'SConscript' -o \
+						-name 'SConstruct' -o \
+						-name 'NEWS' -o \
+						-name 'LICENSE' -o \
+						-name 'COPYING'
```
Comment (mgorny) [inline pkgdiff]:
```suggestion
						-name 'NEWS*' -o \
						-name 'LICENSE*' -o \
						-name 'COPYING*'
```

To cover `.md` etc.

**Comment by mgorny** [comment]
Maybe add cargo.lock too.

File: pkgdiff (hunk 1 above)
Comment (thesamesam) [reply pkgdiff:32]: Meh. Alright. I think I wanted this for some reason but I don't remember what right now. We can revisit if I remember.

**Pull request closed without merging**